### ISSUE TITLE
Release Date should be (in most cases) different from Upload Date

### ISSUE THREAD

**Issue by csnyulas** [ncbo/bioportal-project#25]
Looking at different ontologies in BioPortal, it seems that the Release Date of many ontologies are (displayed as being) set to be the same as their Upload Date.

Here are a few examples:
- [NCIT](http://bioportal.bioontology.org/ontologies/NCIT) (a definite mistake)
- [ICPC-2P](http://bioportal.bioontology.org/ontologies/ICPC2P), [NDDF](http://bioportal.bioontology.org/ontologies/NDDF), [SNOMED-CT](http://bioportal.bioontology.org/ontologies/SNOMEDCT), [MESH](http://bioportal.bioontology.org/ontologies/MESH) (and probably all UMLS files)
- [FMA](http://bioportal.bioontology.org/ontologies/FMA) (and other OWL files)

The reason for this can be multiple. Here are a few that popped into my mind: 
- the release dates and upload dates are in fact recorded as different, but, due to some bug, only one of them is being displayed in both columns of the `Submissions` table
- Release date is not set, and the UI shows the upload date as the release date in such cases. Which is rather confusing than helpful
- The release date is not being properly stored at upload time
- Some auto-suggestion/auto-fill mechanism, or lack of enforcement to fill that field, encourages users to forget to set the proper release date

Whatever the case is, I think this is wrong and should be fixed.

**Comment by jvendetti** [comment]
Part of this issue is a duplicate of #5 (the second bullet point that refers to all UMLS ontologies).

**Comment by csnyulas** [comment]
Indeed,  before posting this issue I searched the issue tracker for any potentially related issues and I found #5. However, while #5 provides a fix for the (existing) UMLS ontologies, my observations suggest that the problem may be more general. I created this separate issue in the hope that the underlying cause of the more general problem can be identified and corrected.

**Comment by jvendetti** [comment]
@csnyulas - yes, I understood that from your initial comments. It's unlikely there will be a single fix to address a solitary underlying cause, due to the fact that UMLS ontologies are imported into BioPortal in a fundamentally different way than all other ontologies submitted by end users. We've already discussed internally how to modify our UMLS import process in such a way that the release date properties for UMLS submission objects are correctly initialized. This will be handled as part of #5.

**Comment by jvendetti** [comment]
Note that "released" is currently a required attribute in the model layer: https://github.com/ncbo/ontologies_linked_data/blob/master/lib/ontologies_linked_data/models/ontology_submission.rb#L42.
